README.md

```diff
@@ -9,12 +9,16 @@
 
 **Jellyseerr** is a free and open source software application for managing requests for your media library. It is a a fork of Overseerr built to bring support for Jellyfin & Emby media servers!
 
+_The original Overseerr has been busy and it's not in their roadmap for the time being for jellyfin/emby support so we started this project as we wanted to bring the overseerr experience to the jellyfin/emby Community!_
```

Comment:
The original Overseerr team have been busy and jellyfin/emby support aren't on their roadmap, so we started this project as we wanted to bring the overseerr experience to the jellyfin/emby Community!